coalib/results/Diff.py

```diff
@@ -244,10 +245,9 @@ def original(self):
         """
         Retrieves the original file.
         """
-        return self._file
+        return self._original
 
-    @property
-    def modified(self):
+    def _raw_modified(self):
```

Comment:
Ah but maybe use in this function instead of `_file` -> `_original`? So then we don't need the real original `_file` any more^^